spec/controllers/project_folders_controller_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ProjectFoldersController, type: :controller do
+  login_user
+  render_views
+
+  let!(:user) { User.first }
```

Comment:
I think explicit `subject.current_user` would be better

Reply:
true